src/app.ts

```diff
@@ -4,6 +4,7 @@ const { version: BOT_VERSION } = require('../package.json');
 import { getPricer } from '@pricer/pricer';
 import Pricer, { GetPricerFn } from './classes/Pricer';
 import { loadOptions } from './classes/Options';
+import HttpManager from './classes/HttpManager';
```

Comment:
What about importing only if enabled, to reduce memory usage slightly, but this is not that important

Reply:
ES6 and TS don't allow importing with `import from` within classes.

Reply:
I know, either use `require` or let it be like it is now